dnfdaemon-server/dbus/interfaces/org.rpm.dnf.v0.Goal.xml

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
-"http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+"file:///usr/share/xml/dbus-1/introspect.dtd">
```

Comment:
On Fedora 35, the file "/usr/share/xml/dbus-1/introspect.dtd" is included in the "dbus-devel" package.
Does this change mean that the user must install the development package on the system in order to use dnfdaemon or not?

Reply:
No, it would be a build dependency. Also, we are not entirely sure if it will be possible to validate xml files containing doc stings. This pr only removes a warning raised by `xsltproc`

Reply:
FWIW, you should be able to switch the dtd validation off using `--novalid` xsltproc switch.